Source/WebCore/platform/graphics/cg/ImageBufferCGBackend.h

```diff
@@ -41,14 +41,17 @@ class WEBCORE_EXPORT ImageBufferCGBackend : public ImageBufferBackend {
     static unsigned calculateBytesPerRow(const IntSize& backendSize);
     static constexpr bool isOriginAtBottomLeftCorner = true;
 
+    std::unique_ptr<GraphicsContext> createContext() final;
+    void contextReleased() final;
+
 protected:
     using ImageBufferBackend::ImageBufferBackend;
-    void applyBaseTransform(GraphicsContextCG&) const;
 
     std::unique_ptr<ThreadSafeImageBufferFlusher> createFlusher() override;
+    virtual CGContextRef ensurePlatformContext() = 0;
 
-    bool originAtBottomLeftCorner() const override;
-    mutable std::unique_ptr<GraphicsContextCG> m_context;
+    GraphicsContextCG* m_context { nullptr };
```

Comment:
Can we at least make this a CheckedPtr?

Seems not great that we're relying on ImageBuffer to manually do lifetime management of this pointer.